node/malus/container/malus.Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM debian:bullseye-slim
```

Comment:
is this file used? there is scripts/dockerfiles/malus.Dockerfile as well
what about node/malus/container/build.sh or node/malus/container/Containerfile-cargo-chef

Reply:
In the `ci` we build the `malus` image from `dockerfiles/malus.Dockerfile`, maybe those files are for local usage @drahnr?

Thanks!

Reply:
There should only be one dockerfile

Reply:
Great, I will remove the the one under `node/malus/container` and change the scripts to use the same as `ci` (scripts/dockerfiles/malus.Dockerfile). 

Thanks!